packages/dev/parcel-namer-s2/S2Namer.js

```diff
@@ -16,6 +16,28 @@ const path = require('path');
 module.exports = new Namer({
   name({bundle}) {
     let mainAsset = bundle.getMainEntry();
+
+    if (!mainAsset?.filePath.includes('react-spectrum/packages')) {
```

Comment:
This is the same logic as below? Could the if conditions be merged?

Reply:
yeah, so open question about this due to ask for non-20 sized icons
I had it originally separate though as a reminder to see if there was a better condition we could use